.github/workflows/release-code-freeze.yml

```diff
@@ -163,15 +164,61 @@ jobs:
         run: |
           pnpm utils code-freeze version-bump ${{ needs.prepare-for-code-freeze.outputs.nextTrunkVersion }}-dev -o ${{ github.repository_owner }} -n ${{ github.event.repository.name}} -b trunk
 
-      # Notify Slack.
+  build-zip:
+    name: 'Build WooCommerce Release Zip'
+    uses: ./.github/workflows/release-build-zip-file.yml
+    needs: [prepare-for-code-freeze, run-code-freeze]
+    with:
+      skip_verify: true
+      create_github_release: false
+      branch: ${{ needs.prepare-for-code-freeze.outputs.nextReleaseBranch }}
+
+  modify-zip-version:
+    name: 'Update version to Dev'
+    runs-on: ubuntu-latest
+    needs: [build-zip, prepare-for-code-freeze]
+    env:
+      DEV_VERSION: ${{ needs.prepare-for-code-freeze.outputs.nextTrunkVersion }}-dev
+    outputs:
+      artifact_url: ${{ steps.upload-modified-zip.outputs.artifact-url }}
+    steps:
+      - name: Download original zip artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: woocommerce
+          path: ./download
+
+      - name: Modify version in woocommerce.php
+        run: |
+          sed -i "s/\* Version:.*/\* Version: $DEV_VERSION/" ./download/woocommerce/woocommerce.php
```

Comment:
There's another version number that needs to be modified [here](https://github.com/woocommerce/woocommerce/blob/85a7bccd3d3c1344523d446476015bd621f2d19c/plugins/woocommerce/includes/class-woocommerce.php#L47).

Could we use `pnpm utils code-freeze version-bump` to bump the version so that we consolidate behind one process?

Reply:
@joshuatf, thanks for the review, it made me realize the version was actually wrong 🤦 I was using the "next trunk version", but it should be the "next release version".
Also, the `version-bump` command is executed in this same flow, some steps above, and it creates a branch (and PR) like `prep/release/x.y-for-next-dev-cycle-x.y`, and that's the one we should use and just replace the `-rc.1` with `-dev`. Does it make sense? 🤔